src/App.tsx

```diff
@@ -58,6 +58,14 @@ function App() {
           <Route path="/communities">
             <Communities />
           </Route>
+          <Route
+            path="/bug"
+            component={() => {
+              window.location.href =
+                'https://github.com/OpenHospitalityNetwork/sleepy.bike/issues/new'
```

Comment:
Interesting way to work around the limitations of the config.

Typically, i'd expect the outside links to be done with `<a href="...">`. May have some UX benefits (e.g. user knows where the link goes when they hover it). I understand that it is done this way because the config allows only internal links.